partners.json

```
@@ -825,5 +825,25 @@
         "height": 84,
         "width": 290,
         "alt": "Logo du partenaire Roannais Agglomération"
+    },
+    {
+        "name": "Communauté d'Agglomération du Bassin de Brive",
```

```suggestion
        "name": "Communauté d’Agglomération du Bassin de Brive",
```
il y a également d’autres occurrences à corriger.